trusted-committer/es/01-introduction-es.asciidoc

```diff
@@ -3,34 +3,34 @@
-para al final conseguir contribuciones sobre la línea de meta.
-El rol de Trusted Committer es tanto demandante como gratificante.
+para tutelar a contribuidores,
+para, en última instancia, lograr que las contribuciones lleguen a término.
+El rol de Trusted Committer es tanto exigente como gratificante.
 Es más que solo ser un portero obstinado.
 Es importante para el éxito de cualquier comunidad InnerSource.
 
-Generalmente hablando, el rol de Trusted Commiter está definido por sus responsabilidades, en lugar de sus privilegios.
-En un nível muy elevado, los Trusted Commiters representan los intereses tanto de su comunidad InnerSource como de los productos que la comunidad está construyendo.
-Se tienen que preocupar de la salud de la comunidad y del producto.
+Generalmente hablando, el rol de Trusted Commiter se define por sus responsabilidades, en lugar de por sus privilegios.
+A muy alto nível, los Trusted Commiters representan los intereses tanto de su comunidad InnerSource como de los productos que la comunidad está construyendo.
+Se preocupan por la salud de la comunidad y del producto.
 Por lo tanto, un Trusted Committer tiene responsabilidades técnicas y orientadas a la comunidad.
 Vamos a explorar ambas en las siguientes secciones.
 
 Antes de entrar en detalle a lo que un Trusted Commiter hace realmente,
 vamos a usar nuestro tiempo comparando el rol de Trusted Committer con otros roles en InnerSource en un nível alto de abstacción
-y explicar por que pensamos que el nombre es apto e importante.
+y explicar por qué pensamos que el nombre es apto e importante.
 Empecemos con el rol de https://innersourcecommons.org/learn/learning-path/contributor/01[Contribuidor].
 Un _Contribuidor_, como su nombre lo indica, hace contribuciones a una comunidad InnerSource.
 Estas contribuciones pueden ser artefactos de código o no-código,
-como reportes de bugs, petición de características, o documentación.
+como informes de defectos, petición de características, o documentación.
 
 _Contribuidores_ pueden ser o no parte de la comunidad.
-Ellos podrían ser enviados por otro equipo para desarrollar una característica que el equipo necesite.
+podrían ser enviados por otro equipo para desarrollar una característica que el equipo necesite.
 Debido a esto a veces nos referimos a los _Contribuidores_ como _Invitados_ o como parte de un _Equipo Invitado_.
 El _Contribuidor_ es responsable de "encajar" y de adaptarse a los procesos y las expectativas de la comunidad.
 
-El _Trusted Committer_ es siempre un miembro de la comunidad InnerSource.
-El cual también se suele referir como _Equipo Anfitrión_.
-En esta analogía el Trusted Committer es responsable de construir una casa y de definir las reglas de la casa,
+El _Trusted Committer_ es siempre un miembro de la comunidad InnerSource,
+a la cual se suele referir como _Equipo Anfitrión_.
+En esta analogía el Trusted Committer es responsable de limpiar una casa y de definir las reglas de la casa,
```

Comment:
limpiar **la** casa y **definir sus reglas**. Determined, since it refers to "anfitrión". Avoid referring to "la casa" twice in the same sentence.